Framework/PythonInterface/core/CMakeLists.txt

```diff
@@ -1,54 +1,63 @@
-###########################################################################
-# Core Python/C++ code used by all layers accessing the Python C API.
-# It contains functionality such as classes for GIL management, conversion
-# of Python errors to exceptions etc.
-#
+# Core Python/C++ code used by all layers accessing the Python C API. It
+# contains functionality such as classes for GIL management, conversion of
+# Python errors to exceptions etc.
+
 # It should not link to any other Mantid libraries.
-###########################################################################
-set ( SRC_FILES
-  src/Converters/NDArrayTypeIndex.cpp
-  src/Converters/WrapWithNDArray.cpp
-  src/ErrorHandling.cpp
-  src/GlobalInterpreterLock.cpp
-  src/NDArray.cpp
-  src/ReleaseGlobalInterpreterLock.cpp
-  src/WrapperHelpers.cpp
-)
-
-set ( INC_FILES
-  inc/MantidPythonInterface/core/Converters/NDArrayTypeIndex.h
-  inc/MantidPythonInterface/core/Converters/VectorToNDArray.h
-  inc/MantidPythonInterface/core/Converters/WrapWithNDArray.h
-  inc/MantidPythonInterface/core/CallMethod.h
-  inc/MantidPythonInterface/core/DllConfig.h
-  inc/MantidPythonInterface/core/ErrorHandling.h
-  inc/MantidPythonInterface/core/GlobalInterpreterLock.h
-  inc/MantidPythonInterface/core/NDArray.h
-  inc/MantidPythonInterface/core/ReleaseGlobalInterpreterLock.h
-  inc/MantidPythonInterface/core/VersionCompat.h
-  inc/MantidPythonInterface/core/WrapperHelpers.h
-)
+
+set(SRC_FILES
+    src/Converters/NDArrayTypeIndex.cpp
+    src/Converters/WrapWithNDArray.cpp
+    src/ErrorHandling.cpp
+    src/GlobalInterpreterLock.cpp
+    src/NDArray.cpp
+    src/ReleaseGlobalInterpreterLock.cpp
+    src/WrapperHelpers.cpp)
+
+set(INC_FILES
+    inc/MantidPythonInterface/core/Converters/NDArrayTypeIndex.h
+    inc/MantidPythonInterface/core/Converters/VectorToNDArray.h
+    inc/MantidPythonInterface/core/Converters/WrapWithNDArray.h
+    inc/MantidPythonInterface/core/CallMethod.h
+    inc/MantidPythonInterface/core/DllConfig.h
+    inc/MantidPythonInterface/core/ErrorHandling.h
+    inc/MantidPythonInterface/core/GlobalInterpreterLock.h
+    inc/MantidPythonInterface/core/NDArray.h
+    inc/MantidPythonInterface/core/ReleaseGlobalInterpreterLock.h
+    inc/MantidPythonInterface/core/VersionCompat.h
+    inc/MantidPythonInterface/core/WrapperHelpers.h)
```

Comment:
Has anyone tried how these changes play together with our Python scripts for creating (and removing classes), e.g., `buildconfig/class_maker.py`?

Reply:
it adds them in the wrong place, it just adds a new section to the end of the file with the old spacing, 
```
set ( INC_FILES
	inc/MantidPythonInterface/core/newtestclass.h
)
set ( SRC_FILES
	src/newtestclass.cpp
)
```
but changing that to follow the correct spacing should probably be a separate issue.

Reply:
@mantidproject/gatekeepers Is anyone aware of this? As it is now the project would not compile anymore if someone uses that script?

Reply:
Good point. I've never really gauged how widely those scripts are used. I would say that one benefit of using the formatting tool is that the scripts will be able to guarantee the formatting of the files when parsing it.

Should we merge this and then @alicerussell1 would you be willing to look at fixing the tool? I suspect it is in [cmakelists_utils.py](https://github.com/mantidproject/mantid/blob/master/buildconfig/cmakelists_utils.py#L32)

Reply:
Yes I can look into this, I'll create an issue for it now.

Reply:
Thanks!